static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.js

```diff
@@ -0,0 +1,214 @@
+// @flow
+import config from 'lib/config';
+import { local } from 'lib/storage';
+import { Message } from 'common/modules/ui/message';
+import { commercialFeatures } from 'commercial/modules/commercial-features';
+import mediator from 'lib/mediator';
+import { testCanBeRun } from 'common/modules/experiments/test-can-run-checks';
+import MembershipEngagementBannerTests from 'common/modules/experiments/tests/membership-engagement-banner-tests';
+import { inlineSvg } from 'common/views/svgs';
+import { isInTest, variantFor } from 'common/modules/experiments/segment-util';
+import { epicEngagementBannerTests } from 'common/modules/experiments/acquisition-test-selector';
+import membershipEngagementBannerUtils from 'common/modules/commercial/membership-engagement-banner-parameters';
+import { isBlocked } from 'common/modules/commercial/membership-engagement-banner-block';
+import ophan from 'ophan/ng';
+import { get } from 'lib/geolocation';
```

Comment:
The named import makes a bit unhappy. What about `import { get as getGeoLocation } from 'lib/geolocation'`?